charts/firecloudorch/templates/deployment.yaml

```diff
@@ -157,6 +157,9 @@ spec:
         livenessProbe:
           {{- toYaml .Values.probes.liveness.spec | nindent 10 }}
         {{- end }}
+        startProbe:
```

Comment:
spotted a typo here: should be `startupProbe`

Reply:
c1eed21